land_claims.civmap.json

```diff
@@ -72,7 +72,7 @@
 {"name":"Aquaderia\n(Columbia)","color":"#ffdc17","declutter":3,"id":"5c695402-1737-430c-a8e4-612eb8fe4360","polygon":[[[-11539,-836],[-11591,-860],[-11603,-912],[-11651,-916],[-11695,-892],[-11691,-856],[-11751,-832],[-11823,-848],[-11867,-860],[-11907,-864],[-11975,-808],[-11963,-716],[-12027,-600],[-12099,-316],[-12051,-276],[-12051,-232],[-11579,-320],[-11591,-376],[-11587,-456],[-11563,-516],[-11567,-548],[-11515,-616],[-11491,-660],[-11519,-816]]]},
 {"name":"Armenia\n(The Empire)","shortname":"(EM)","color":"#333333","declutter":4,"id":"c3521388-c47d-4215-a38f-d283481a4131","polygon":[[[-8645,-1800],[-8761,-1736],[-8873,-1644],[-8985,-1516],[-9001,-1428],[-9161,-1372],[-9185,-1188],[-9001,-740],[-8977,-480],[-8845,-448],[-8837,-348],[-8821,-312],[-8753,-332],[-8705,-412],[-8633,-376],[-8269,-312],[-8081,-308],[-7981,-396],[-8065,-568],[-8037,-636],[-7977,-656],[-7961,-728],[-8001,-732],[-8017,-782],[-8035,-805],[-8070,-791],[-8104,-851],[-8099,-868],[-8109,-880],[-8148,-882],[-8205,-972],[-8113,-1072],[-8057,-1108],[-8001,-1160],[-7881,-1284],[-7837,-1300],[-7841,-1384],[-7953,-1388],[-8013,-1460],[-8089,-1428]]]},
 {"name":"Arrelani\n(Caledonia)","shortname":"Arrelani","color":"#15316a","declutter":3,"x":2430,"z":780,"id":"8f94d787-8751-4b50-a4b2-ff26558ef4d3","polygon":[[[2462,1252],[2516,1176],[2550,1094],[2576,1068],[2576,993],[2570,992],[2568,992],[2568,976],[2560,976],[2560,959],[2620,884],[2706,848],[2742,812],[2776,732],[2806,698],[2768,624],[2756,596],[2686,560],[2708,414],[2664,363],[2661,362],[2586,368],[2466,366],[2412,393],[2349,338],[2334,327],[2311,311],[2358,392],[2332,407],[2288,409],[2270,432],[2216,458],[2182,516],[2186,609],[2142,718],[2084,803],[2109,901],[2174,933],[2225,992],[2221,1039],[2244,1055],[2257,1115],[2243,1144],[2306,1273],[2346,1259],[2397,1224],[2449,1242],[2463,1250]]]},
-{"name":"Aurum/n(varkonia)","shortname":"Aurum","color":"#0a43be","declutter":3,"x":2930,"z":-350,"id":"dbdebb67-ebf4-49ec-bbed-1ab7e483c38f","polygon":[[[2203,-620],[2246,-670],[2317,-693],[2335,-739],[2399,-760],[2552,-757],[2715,-614],[3350,-612],[3393,-569],[3422,-536],[3479,-530],[3477,-494],[3542,-421],[3524,-400],[3484,-364],[3454,-287],[3476,-250],[3474,-195],[3525,-136],[3425,-59],[3099,16],[3078,18],[3017,70],[2950,50],[2936,-32],[2824,-88],[2676,-133],[2535,-247],[2453,-309],[2309,-345],[2261,-429],[2239,-473],[2226,-558],[2203,-617]]]},
+{"name":"Aurum\n(varkonia)","shortname":"Aurum","color":"#0a43be","declutter":3,"x":2930,"z":-350,"id":"dbdebb67-ebf4-49ec-bbed-1ab7e483c38f","polygon":[[[2203,-620],[2246,-670],[2317,-693],[2335,-739],[2399,-760],[2552,-757],[2715,-614],[3350,-612],[3393,-569],[3422,-536],[3479,-530],[3477,-494],[3542,-421],[3524,-400],[3484,-364],[3454,-287],[3476,-250],[3474,-195],[3525,-136],[3425,-59],[3099,16],[3078,18],[3017,70],[2950,50],[2936,-32],[2824,-88],[2676,-133],[2535,-247],[2453,-309],[2309,-345],[2261,-429],[2239,-473],[2226,-558],[2203,-617]]]},
```

Comment:
lovercase varkonia still but fine by me for now

Reply:
I thought I did in the unnamed commit?

Reply:
Oh different Varkonia